test/widgets/action_sheet_test.dart

```diff
@@ -1085,6 +1086,139 @@ void main() {
       });
     });
   });
+
+  group('channel action sheet', () {
+    late ZulipStream someChannel;
+    late PerAccountStore store;
+
+    Future<void> prepare({UnreadMessagesSnapshot? unreadMsgs}) async {
+    final stream = eg.stream();
+    someChannel = stream;
+    addTearDown(testBinding.reset);
+
+    unreadMsgs ??= eg.unreadMsgs(channels: [
+      eg.unreadChannelMsgs(
+        streamId: stream.streamId,
+        topic: 'topic',
+        unreadMessageIds: [1],
+      ),
+    ]);
+
+    await testBinding.globalStore.add(eg.selfAccount, eg.initialSnapshot(
+      realmUsers: [eg.selfUser, eg.otherUser],
+      streams: [someChannel],
+      subscriptions: [eg.subscription(someChannel)],
+      unreadMsgs: unreadMsgs));
+    store = await testBinding.globalStore.perAccount(eg.selfAccount.id);
+    connection = store.connection as FakeApiConnection;
+  }
```

Comment:
Please read through the indentation in this file and make it consistent with the project style; this part (and at least one other part) is currently hard to read.